plans/dht/utils/create-dht-node.go

```diff
@@ -0,0 +1,44 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/ipfs/testground/sdk/runtime"
+
+	datastore "github.com/ipfs/go-datastore"
+	libp2p "github.com/libp2p/go-libp2p"
+	host "github.com/libp2p/go-libp2p-core/host"
+	kaddht "github.com/libp2p/go-libp2p-kad-dht"
+	dhtopts "github.com/libp2p/go-libp2p-kad-dht/opts"
+)
+
+// CreateDhtNode creates a libp2p Node and a DHT on top of it
+func CreateDhtNode(ctx context.Context, runenv *runtime.RunEnv) (host.Host, *kaddht.IpfsDHT, error) {
+	// Test Parameters
+	var (
+		bucketSize = runenv.IntParamD("bucket_size", 20)
+		randomWalk = runenv.BooleanParamD("random_walk", false)
+	)
+
+	node, err := libp2p.New(ctx)
+	if err != nil {
+		runenv.Abort(err)
```

Comment:
We can move the Abort to the caller. These Aborts here and below are redundant, since the caller Aborts when we return an error. That also removes the "spooky action at a distance" effect, such that the code of the test (and not the utils) are the thing that print outputs on the runenv.